lib/storage.dart

```diff
@@ -23,13 +24,14 @@ class Storage {
 
   _init() async {
     final appDir = await path_provider.getApplicationDocumentsDirectory();
-    final documentPath = appDir.path;
-    const folderName = "PilotS3";
-    final path = Directory("$documentPath\\$folderName");
-    if (!(await path.exists())) {
-      path.create();
+    final documentPath =
+        Platform.isLinux ? path.join(appDir.path, '.config') : appDir.path;
+    final folderName = Platform.isLinux ? "pilot-s3" : "PilotS3";
+    final configPath = Directory(path.join(documentPath, folderName));
+    if (!(await configPath.exists())) {
+      configPath.create(recursive: true);
```

Comment:
May be remove path_provider then?

Reply:
`path_provider` provide method  `getApplicationDocumentsDirectory` for home dir getting
`path` used just for join strings with os specific slash

both plugin needed i think

Reply:
kk